### PR TITLE
add tutorial doc for semantic search on amazon opensearch

### PR DESCRIPTION
**Pull request by ylwu-amzn**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/tutorials/aws/semantic_search_with_cohere_embedding_model.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+# Steps
+
+## 0. Create OpenSearch cluster
+
+Go to AWS OpenSearch console UI and create OpenSearch domain.
+
+Copy the domain ARN which will be used in later steps.
+
+## 1. Create secret
```
Comment (Zhangxunmt) [inline docs/tutorials/aws/semantic_search_with_cohere_embedding_model.md:21]: Should we mention in the Topics that this cohere model is available in bedrock so we can basically use the same steps for the bedrock service to run semantic search using the cohere model?

**Review by jngz-es — APPROVED** [review]

**Review by Zhangxunmt — APPROVED** [review]

File: docs/remote_inference_blueprints/sagemaker_connector_blueprint.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
 ### Sagemaker connector blueprint example for embedding:
 
+Read more details on https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/blueprints/
+
+Make sure your Sagemaker model input follow such format, so the [default pre-process function](https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/blueprints/#preprocessing-function) can work
```
[the remark this answers is not in the document]
Comment (ylwu-amzn) [reply docs/remote_inference_blueprints/sagemaker_connector_blueprint.md:5]: sure, good point

File: docs/remote_inference_blueprints/sagemaker_connector_blueprint.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+```
+["hello world", "how are you"]
+```
+and output follow such format, so the [default post-process function](https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/blueprints/#post-processing-function) can work
```
[the remark this answers is not in the document]
Comment (ylwu-amzn) [reply docs/remote_inference_blueprints/sagemaker_connector_blueprint.md:9]: sure, good point

File: docs/tutorials/aws/semantic_search_with_cohere_embedding_model.md (hunk 1 above)
Comment (ylwu-amzn) [reply docs/tutorials/aws/semantic_search_with_cohere_embedding_model.md:21]: sure

**Review by jngz-es — APPROVED** [review]

**Review by Zhangxunmt — APPROVED** [review]

**Pull request merged**